cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/ckan/CKANBackendImpl.java

```diff
@@ -440,10 +440,11 @@ private boolean existsView(String resId) throws Exception {
         } // try catch
```

Comment:
CNR entry missing?

(Maybe it is already included, but as you use to include CNR entry in each very PR, better to check ;)